save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/SelectForm.kt

```diff
@@ -64,6 +67,16 @@ external interface SelectFormRequiredProps<D : Any> : Props {
      */
     var notFoundErrorMessage: String?
 
+    /**
+     * Add custom elements under the form label in order to create new item.
+     */
+    var addNewItemChildrenBuilder: ((ChildrenBuilder) -> Unit)?
```

Comment:
maybe to replace to avoid `with
```suggestion
    var addNewItemChildrenBuilder: (ChildrenBuilder.() -> Unit)?
```

Reply:
As far as it is external interface, I cannot use lambdas with receiver (seems like unless it is suspend)
<img width="541" alt="screenshot" src="https://user-images.githubusercontent.com/35039155/189878155-071a1f5e-1dad-49c2-9242-1d3b94e8bf9d.png">

Reply:
interesting, then okay